src/agr/app.py

```diff
@@ -30,6 +31,12 @@ def setup_logging(logger_name):
 def main(generated_files_folder='generated_files',
          fasta_sequences_folder='sequences',
          skip_chromosomes={'Unmapped_Scaffold_8_D1580_D1567'}):
+    #  generate_vcf_files(generated_files_folder, fasta_sequences_folder, skip_chromosomes)
+    generate_orthology_file(generated_files_folder, alliance_db_version)
```

Comment:
This I will investigate further (the "seeming to run twice issue and call to `exit`" here as per your comment on the PR).